crates/rpc/rpc/src/debug.rs

```diff
@@ -891,7 +891,11 @@ where
             .to_rpc_result()?
             .unwrap_or_default()
             .into_iter()
-            .map(|receipt| receipt.with_bloom().envelope_encoded())
+            .map(|receipt| {
+                let mut out = vec![];
+                receipt.with_bloom().encode_2718(&mut out);
```

Comment:
this can be done with a `encoded_2718` with returns the vec

Reply:
thank you, i just update